yarn.lock

```diff
@@ -193,6 +193,7 @@
   "integrity" "sha512-WBwekcqacdY2e9AF/Q7WLFUWmdJGJTkbjqTjoMDgXkVZ3ZRUvOPsLb5KdwISoQVsbP+DQzVZW4Zhci0DvpbNTQ=="
   "resolved" "https://registry.npmjs.org/@babel/runtime/-/runtime-7.16.3.tgz"
   "version" "7.16.3"
+<<<<<<< HEAD
```

Comment:
Hi John, 

I think this is produced by a conflict in git , it need to be resolved before merge.  :) 

Thanks!